Merge Updates - ETAC, Lighting Mods, ASLAL: NB & Verdant OCS

MacSplody opened the pull request:
[New] Expanded Towns and Cities (#282)
Add more incompatible messages for lighting mods (#280)
[NEW] Alternate Start -- New Beginnings (#285)
Flag latest Verdant OCS version as being Incompatible with Open Cities. (#287)

MacSplody committed:
Add more incompatible messages for lighting mods (#280)

MacSplody committed:
[New] Expanded Towns and Cities (#282)

MacSplody committed:
[NEW] Alternate Start -- New Beginnings (#285)

Group
- Move to alt start group.

Bash tags
- Remove bad tag 'Sound'
- Add 'Scripts' Tags for vmad change.

Cleaning info
- Clean plug-in.

MacSplody committed:
Flag latest Verdant OCS version as being Incompatible with Open Cities. (#287)

* Verdant - OCS version Incompatible with Open Cites

#286

* fix syntax

Freso commented on masterlist.yaml:
```diff
@@ -4190,6 +4212,315 @@ plugins:
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/13552']
+    msg:
+      - type: say
+        content: 'Plug-ins Masters require sorting.'
```
I'd say "Plugin’s". We generally use "plugin" without the dash elsewhere in the documentation, and the apostrophe is needed to denote genitive… otherwise we're talking about multiple plugins and then the sentence kind of doesn't make sense anymore. :)

MacSplody replied:
Done.

Freso commented on masterlist.yaml:
```diff
@@ -4190,6 +4212,315 @@ plugins:
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/13552']
+    msg:
+      - type: say
+        content: 'Plug-ins Masters require sorting.'
```
If this is repeated, it seems like it should be made an anchor in `commons`.

MacSplody replied:
Okay I cleaned up the grammar, not my strong suit and added the message to `commons`

Freso reviewed (changes requested):
Most of the changes here I don't really have an idea about, since I don't know the plugins involved at all, but the logic itself generally seems sane.

Only think I'd want to see here is a small wording (or rather grammar) change, and maybe having the involved `msg` moved up to the `commons` section.

MacSplody committed:
Replace 'Plug-ins Masters require sorting' with common entry

Language updated to now read as:
- Plugin''s Masters require sorting.

MacSplody commented:
Okay I cleaned up the grammar, not my strong suit and added the message to `commons`

MacSplody committed:
[Correction] Util keys

Remove redudant util keys from dirty plugin entries.

Wrap Util version string in apostrophes.

MacSplody committed:
Format fixes.

- Add leading space to crc in checksum.
- Add leading & trailing space to URLs wrapped in square brackets.

Freso reviewed (approved):
Looks good to me now, but again, I can't comment much on how it'll affect the game, since I haven't done Skyrim since before SSE was a thing. Syntactically it looks fine though. 👍
